### ISSUE TITLE
Fix incorrect capitalization of acronyms journal names

### ISSUE THREAD

**Issue by salutis** [andras-simonyi/citeproc-el#80]
# Example

```
@Article{carter+2015,
  author       = {S.E. Carter and M. Jones and V.F. Gladwell},
  title        = {Energy expenditure and heart rate response to
                  breaking up sedentary time with three different
                  physical activity interventions},
  journal      = {Nutrition, metabolism, and cardiovascular diseases,
                  NMCD},
  year         = 2015,
  volume       = 25,
  doi          = {10.1016/j.numecd.2015.02.006},
  number       = 5,
  abstract     = {...}
}
```

(CSL: APA)

# Expected

Carter, S., Jones, M., & Gladwell, V. (2015). Energy expenditure and heart rate response to breaking up sedentary time with three different physical activity interventions. _Nutrition, Metabolism, and Cardiovascular Diseases, **NMCD**, 25_(5). https://doi.org/10.1016/j.numecd.2015.02.006

[bold emphasis: mine]

# Actual

Carter, S., Jones, M., & Gladwell, V. (2015). Energy expenditure and heart rate response to breaking up sedentary time with three different physical activity interventions. _Nutrition, Metabolism, and Cardiovascular Diseases, **Nmcd**, 25_(5). https://doi.org/10.1016/j.numecd.2015.02.006

[bold emphasis: mine]

**Comment by andras-simonyi** [comment]
you need to use curly brackets indicating regions where the original case has to be kept:
``` bibtex
journal      = {Nutrition, metabolism, and cardiovascular diseases, {NMCD}},
```

**Comment by salutis** [comment]
@andras-simonyi Even in journal names? I thought the processors leave those alone, like publishers.

**Comment by andras-simonyi** [comment]
Of course, the concrete transformation depends on the style and processor, but for instance Pandoc renders the original 'Journal of Capitalized lowercase' journal field as 'Journal of Capitalized Lowercase'' with the CSL chicago-author-date style, so yes, processors treat journal names similarly to other titles in this respect.

**Comment by salutis** [comment]
@andras-simonyi I have just tried `pandoc` (2.16.2) with the same APA CSL, and it does re-capitalize the title to sentence case (like `citeproc.el` does) which proves my statement above ("leave those alone") wrong. As for the "NMCD", `pandoc` _does_ generate "NMCD" instead of "Ncmd" even without braces. I guess we could consider that a bug (or feature) in `pandoc`. In fact, I have just tried to add "NCMD" into the title, and `pandoc` generates "NCMD" for that as well. Somewhere along the way, `pandoc` has learned not to downcase all-uppercase words!

**Comment by salutis** [comment]
Closing. Thank you @andras-simonyi for helping me to see the facts.
